lib/internal/bootstrap_node.js

```diff
@@ -270,7 +270,12 @@
 
   function installInspectorConsole(globalConsole) {
     const wrappedConsole = NativeModule.require('console');
-    const inspector = process.binding('inspector');
+    var inspector;
+    try {
+      inspector = process.binding('inspector');
+    } catch (ignored) {
+      return wrappedConsole;
+    }
     const config = {};
     for (const key of Object.keys(wrappedConsole)) {
       if (!globalConsole.hasOwnProperty(key))
```

Comment:
The error message suggests that `globalConsole` is `undefined`, right? Shouldn't the fix be installing a guard for it?